CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,207 @@
+
+#### Approver exclusion
+
+Approvers on any specialty may never be interim. This is to prevent abuse of the interim role system, and to consolidate the needed trust between the other maintainers & approvers for the different elements of the codebase.
+
+### Limits
+- There can only be a maximum of two interim roles per specialty (two CI members; two CI reviewers; two CI interim approvers; two provider Members; two provider Reviewers; two provider interim Approvers).
+
+### Examples
+- **Interim Member**: Has fewer than 8 substantive contributions but commits to achieve them within 3 months.
+- **Interim Reviewer**: Has not yet reviewed 20+ PRs but is actively being mentored to do so.
+
+### Process
+1. Maintainers discuss and vote on the need for an interim role (lazy consensus).
+2. Scope and duration are defined clearly (specialty, responsibilities, expected milestones).
+3. Nomination of interim roles is done by lazy consensus.
+4. Interim roles are granted for a maximum of three months.
+5. Interim status is reviewed at the end of the period:
+   - If requirements are met → promotion to the permanent role.
+   - If not met → revert to previous role; contributor may try again later.
+
+### Handling abuse
+
+Interim roles are not a substitute for permanent roles. If a contributor is abusing the interim role system, they may be demoted to their previous role.
+
+---
+
+## Advancement Process
+
+1. **Nomination** by an eligible community member (Member+).
+2. **Sponsorship** by two role holders at the **target level or higher** (within the specialty where applicable).
+3. **Review** of activity and behavior (quality, reliability, collaboration, responsiveness).
+4. **Decision** by lazy consensus of the relevant group (or **supermajority** if contested).
+
+---
+
+## Inactivity
+
+A role holder may be considered inactive if they have not actively contributed in their specialty or general project responsibilities for **six (6) consecutive months**.
+
+### Measurement Sources
+- GitHub activity: merged PRs, PR reviews, issue triage/comments.
+- Participation in community calls or asynchronous design discussions.
+- Specialty-specific metrics (e.g., CI job maintenance, provider updates, security disclosures).
+
+### Triggering Process
+1. **Detection**  
+   - Activity is reviewed at least quarterly by Maintainers or via automation.  
+   - Any Maintainer may propose an inactivity review for a role holder.
+2. **Notification**  
+   - A public issue is opened in a `community`/`governance` space (or email if sensitive).  
+   - The individual is tagged/emailed and given **30 days** to respond.
+3. **Grace Period**  
+   - If the contributor indicates intent to return, no change is made.  
+   - If there is no response or no significant activity within the grace period, proceed.
+4. **Decision**  
+   - Demotion is decided by **lazy consensus** of Maintainers, or **supermajority** if contested.
```

Comment:
do e.g. approvers get demoted to `member` status or kicked out of the organisation? I guess that should be made clear here ;)